politeiawww/api/v1/v1.go

```diff
@@ -250,6 +246,10 @@ var (
 	// PoliteiaWWWAPIRoute is the prefix to the API route
 	PoliteiaWWWAPIRoute = fmt.Sprintf("/v%v", PoliteiaWWWAPIVersion)
 
+	// PolicyMaxMDs is the maximum number of markdown files accepted
+	// when creating a new proposal
+	PolicyMaxMDs uint
```

Comment:
Does this belong here?

Reply:
It is where we keep all other policy settings, so yes.